src/configs/rollup.es6.config.js

```diff
@@ -37,8 +37,15 @@ const minify = require('rollup-plugin-terser').terser
 const license = require('rollup-plugin-license')
 const os = require('os')
 const extensions = ['.js', '.ts']
+const deepMerge = require('deepmerge')
 
-module.exports = {
+let customConfig
+
+if (process.env.LNG_CUSTOM_ROLLUP === 'true') {
+  customConfig = require(path.join(process.cwd(), 'rollup.es6.config'))
```

Comment:
What about the _env_?

Reply:
The ones that are mentioned in the env will be ignored here and the custom config takes the priority here.